### ISSUE TITLE
Provide py-afl-cmin

### ISSUE THREAD

**Issue by jabdoa2** [jwilk/python-afl#6]
Would it be possible to have a python wrapper for afl-cmin as well? This would be very useful for effective fuzzing.

**Comment by jwilk** [comment]
So bare afl-cmin doesn't work, because it naively checks the binary for instrumentation:
``` 
[-] Error: binary '/usr/bin/python' doesn't appear to be instrumented.
```
I added `py-afl-cmin` in 5b773d706015b7d4a53d60ff0407aa21b35e632b.

**Comment by jwilk** [comment]
Fixed in python-afl 0.6.

**Comment by jabdoa2** [comment]
Thanks
